lib/model/account/AccountCtrl.dart

```diff
@@ -19,8 +19,8 @@ class AccountCtrl {
   final StorageService _storage;
 
   AccountCtrl(this._jsApi, this._storage, this._accountModel) {
-    _initSavedDeviceAccountAddress(_storage);
     _initJsObservables(_jsApi, _storage);
+    _initSavedDeviceAccountAddress(_storage);
```

Comment:
observables need to be registered first otherwise account can't be registered when set